src/components/datepicker/datepicker-next-day/readme.md

```diff
@@ -28,6 +28,9 @@ if it is disabled, or if the selected date is equal to the input's `max` attribu
 </sbb-form-field>
 ```
 
+Since the component needs the `sbb-datepicker` to work properly, if they are dynamically added,
+both standalone or within the `sbb-form-field`, they must have the same parent element to be correctly connected.
+
```

Comment:
Since the 'dynamically load' thing is hard to understand and explain, is it acceptable to inform the consumer that the date picker and the accessories components need to be children of the same element?

Reply:
You mean remove the 'dynamically added' part, and just say 'when you use them, use them as siblings'?

Reply:
Yep, I feel like it would simplify the communication

Reply:
Done!